src/modules/multi-file-upload/components/upload-list/upload.item.component.js

```diff
@@ -309,12 +324,19 @@ export default class UploadItemComponent extends Component {
      * @memberof UploadItemComponent
      */
     createEditLink() {
-        if (!this.state.struct) {
+        const { struct } = this.state;
+
+        if (!struct) {
             return;
         }
 
-        return Routing.generate('ez_content_draft_create', {
-            contentId: this.state.struct.Content._id,
+        const content = struct.Content;
+
+        return window.Routing.generate('ez_content_draft_edit', {
```

Comment:
This should not be done by edit link but should be sent a form to edit. Should be done like is in SubItemList. // cc @webhdx

Reply:
Good catch! :)